gulp/index.js

```diff
@@ -43,22 +45,35 @@ module.exports = (gulp, config) => {
         },
 
         /**
-         * Returns an array of task names of the format [prefix]-[...prefixes]-[project]
-         * for every project with the given block.
-         * @param block  {string} name of project config block
-         * @param prefix {string} first prefix, defaults to block name
-         * @param prefixes {string[]} additional prefixes before project name
-         * @returns {string[]}
+         * Define a group of tasks for projects with the given config block.
+         * The special block `"all"` will operate on all projects.
+         * The `block` is used as the task name, unless `name` is explicitly defined.
+         * The `taskFn` is called for each matched project with `(project, taskName, depTaskNames)`.
+         * The task name is of the format `[name]-[project.id]`.
+         * Finally, a "group task" is defined with the base name that runs all the project tasks.
+         * This group task can be configured to run in parallel or in sequence.
+         * @param {{block: string, name?: string, parallel?: boolean}} options
```

Comment:
block really doesn't mean anything. what do you mean by "config block sounds better than config scope"? also `projectsWithScope` sounds fine to me